admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -139,7 +166,7 @@ export const ApplicationPage: React.FunctionComponent = () => {
                                                 helperText='This is a short name, only composed of a-z 0-9 and - characters.'>
                                                 <TextInput
                                                     type='text'
-                                                    value={ eventType.name }
+                                                    defaultValue={ eventType.name }
```

Comment:
I don't think we need to change this to `defaultValue`. `value` seems fine.

```suggestion
                                                    value={ eventType.name }
```